source/server/config/http/transformation_filter_config_factory.cc

```diff
@@ -21,58 +21,28 @@ namespace Configuration {
 typedef Http::FunctionalFilterMixin<Http::FunctionalTransformationFilter>
     MixedTransformationFilter;
 
-HttpFilterFactoryCb TransformationFilterConfigFactory::createFilterFactory(
-    const Json::Object &, const std::string &, FactoryContext &) {
-  NOT_IMPLEMENTED;
-}
-
-HttpFilterFactoryCb
-TransformationFilterConfigFactory::createFilterFactoryFromProto(
-    const Protobuf::Message &config, const std::string &stat_prefix,
-    FactoryContext &context) {
-  UNREFERENCED_PARAMETER(stat_prefix);
-
-  /**
-   * TODO:
-   * The corresponding `.pb.validate.h` for the message is required by
-   * Envoy::MessageUtil.
-   * @see https://github.com/envoyproxy/envoy/pull/2194
-   *
-   * #include "transformation_filter.pb.validate.h"
-   *
-   * return createFilter(
-   *    Envoy::MessageUtil::downcastAndValidate<const
-   * envoy::api::v2::filter::http::Transformations&>(proto_config), context);
-   * */
+TransformationFilterConfigFactory::TransformationFilterConfigFactory()
+    : FactoryBase(Config::TransformationFilterNames::get().TRANSFORMATION) {}
```

Comment:
Consider moving the constructor's definition to the header file, to be consistent with the other filters.